src/Wellcome.Dds/Utils/Caching/BinaryObjectCache.cs

```diff
@@ -140,6 +140,12 @@ private void PutInMemoryCache(T t, string cacheKey)
             {
                 return;
             }
+
+            if (cacheKey.Contains("b19974760"))
```

Comment:
is it worth pulling this into a constant, or `IsChemistAndDruggist()`, for clarity?